modules/ROOT/pages/dw-core-functions-filter.adoc

```diff
@@ -145,77 +145,6 @@ users filter ((item, index) -> item.error == null)
 ]
 ----
 
-=== Example
-
-This example reads a JSON array and uses the `filter` function to extract the phone numbers that are active.
-
-==== Input
-
-[source,JSON, linenums]
-----
-{
-  "Id": "1184001100000000517",
-  "marketCode": "US",
-  "languageCode": "en-US",
-  "profile": {
-  "base": {
-     "username": "TheMule",
-     "activeInd": "R",
-     "phone": [
-       {
-          "activeInd": "Y",
-          "type": "mobile",
-          "primaryInd": "Y",
-          "number": "230678123"
-       },
-       {
-         "activeInd": "N",
-         "type": "mobile",
-         "primaryInd": "N",
-         "number": ""
-       },
-       {
-          "activeInd": "Y",
-          "type": "mobile",
-          "primaryInd": "Y",
-          "number": "154896523"
-       }
-      ]
-    }
-  }
- }
-----
-
-==== Source
-[source,DataWeave, linenums]
-----
-%dw 2.0
-output application/json
----
-{
-    id: payload.Id,
-    markCode: payload.marketCode,
-    languageCode: payload.languageCode,
-    username: payload.profile.base.username,
-    phoneNumber: (payload.profile.base.phone filter ($.activeInd == "Y" and $.primaryInd== "Y")).number default []
-}
-----
-
-==== Output
-[source,JSON, linenums]
-----
-{
-  "id": "1184001100000000517",
-  "markCode": "US",
-  "languageCode": "en-US",
-  "username": "TheMule",
-  "phoneNumber": [
-    "230678123"
-    "154896523"
-  ]
-}
-----
-
```

Comment:
Do you think this example needs to be added to the data-weave repo? Or was it deleted intentionally?